lib/api/user_api.dart

```diff
@@ -43,7 +44,12 @@ class UserApi {
   Observable<SettingsModel> getSettings(String id) {
     return _http
         .get('/$id/settings')
-        .map((Response res) => SettingsModel.fromJson(res.json['data']));
+        .map((Response res){
+          if (res.json['success'] == false) {
+            throw ApiException(res);
```

Comment:
Seems like the error message is just an error key which some may find difficult to interpret. Also, the ApiException class is left with a TODO.
It's probably out of your scope for this issue, so I am not going to 'not approve' because of this.

Reply:
You are right! We also judged that out of scope, for this current issue. It would appear, that no issues in api_client are talking about this, so you could open an issue about this.